### PR TITLE
add remove dbus object function on bus and update tests

### PR DESCRIPTION
**Pull request by mc18g13**

### CODE REVIEW

File: dasbus/connection.py [hunk 1]
```diff
@@ -286,14 +286,22 @@ def publish_object(self, object_path, obj,
         )
         object_handler.connect_object()
 
-        self._registrations.append(object_handler.disconnect_object)
+        self._registrations[object_path] = object_handler.disconnect_object
+
+    def unregister(self, object_path):
```
Comment (poncovka) [inline dasbus/connection.py:291]: I would prefer to have the `unregister_service` and `unpublish_object` methods to make it more future-proof. Then the logging messages can match their counterparts.

**Review by poncovka — CHANGES_REQUESTED** [review]
Otherwise, it looks great. Thank you!

File: dasbus/connection.py (hunk 1 above)
Comment (mc18g13) [reply dasbus/connection.py:291]: Okay sure... hows that?

Comment (poncovka) [reply dasbus/connection.py:291]: It looks good. Could you also rename the `unregister` method  to `_unregister` to make it private, please? That would be perfect.

**Review by poncovka — CHANGES_REQUESTED** [review]
See https://github.com/rhinstaller/dasbus/pull/74/files#r803897786.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**